frontend/src/Interface/FieldDetailsInterface.tsx

```diff
@@ -6,6 +6,7 @@
  * @author @GDamaso
  */
 interface FieldDetailInterface {
+  id: number;
```

Comment:
.nmp actually do use `Id` keys for `Fields`, but they have a capital `I`. Would you mind changing this to `Id` so it's easier to convert things later on?